### PR TITLE
Update LearningRateScheduler in callbacks.py

### PR DESCRIPTION
**Pull request by 8bitmp3**
See #30002 : Update the description of LearningRateSchedule, add "Example:" to the example, move Arguments below the example

### CODE REVIEW

**Comment by gbaned** [comment]
@8bitmp3 Please sign CLA in order to proceed with next steps. Thank you!

**Comment by 8bitmp3** [comment]
@gbaned signed it, thanks!

**Comment by 8bitmp3** [comment]
just checking in to see if anything else needs to be done @rchao @gbaned thanks!

File: tensorflow/python/keras/callbacks.py [hunk 1]
```diff
@@ -1308,12 +1308,11 @@ def on_epoch_end(self, epoch, logs=None):
...
 class LearningRateScheduler(Callback):
   """Learning rate scheduler.
-
-  Arguments:
-      schedule: a function that takes an epoch index as input
-          (integer, indexed from 0) and returns a new
-          learning rate as output (float).
-      verbose: int. 0: quiet, 1: update messages.
+  
+  A callback for scheduling the learning rate to change after every epoch or batch.
```
Comment (rchao) [inline tensorflow/python/keras/callbacks.py]: The learning rate doesn't necessarily change after every epoch/batch, as it depends on the `schedule` function. Also I don't think it currently supports changing at batch boundaries.

**Review by rchao — CHANGES_REQUESTED** [review]
Thanks for the change!

File: tensorflow/python/keras/callbacks.py (hunk 1 above)
Comment (8bitmp3) [reply tensorflow/python/keras/callbacks.py]: @rchao spot on! How about we change it to:
```
A callback for scheduling to update the optimizer's learning rate at the start of each epoch. The learning rate used in this callback replaces the default learning rate set by the gradient descent optimization algorithm.
```
I am just trying to make sure people do less googling when working the callbacks APIs. Unfortunately, this was not in @fchollet 's excellent Deep Learning with Python with but is covered in @ageron's Hands-On ML with SKlearn, Keras and TF (upcoming v2 of the book).

**Comment by mellanox-github** [comment]
Can one of the admins verify this patch?

File: tensorflow/python/keras/callbacks.py [hunk 2]
```diff
@@ -1308,12 +1308,12 @@ def on_epoch_end(self, epoch, logs=None):
...
   """Learning rate scheduler.
-
-  Arguments:
-      schedule: a function that takes an epoch index as input
-          (integer, indexed from 0) and returns a new
-          learning rate as output (float).
-      verbose: int. 0: quiet, 1: update messages.
+  
+  A callback for scheduling to update the optimizer's learning rate at the start of each epoch. 
+  The learning rate used in this callback replaces the default learning rate set by the gradient descent optimization algorithm.
```
Comment (rchao) [inline tensorflow/python/keras/callbacks.py]: I think the optimization is not necessarily gradient descent. Rephrase it to avoid confusion?

**Review by rchao — CHANGES_REQUESTED** [review]
Thanks for the request!

File: tensorflow/python/keras/callbacks.py (hunk 1 above)
Comment (rchao) [reply tensorflow/python/keras/callbacks.py]: Thanks for the reply!

**Comment by gbaned** [comment]
@8bitmp3 Could you please check reviewer comments and keep us posted. Thanks!

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

**Comment by gbaned** [comment]
I'm going to go ahead and close this PR, because it seems to have stalled. If you're still interested in pursing this (and responding to my comments), please feel free to reopen!
